src/global.css

```diff
@@ -1,32 +1,188 @@
-body{
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  align-content: center;
-  margin:auto;
-  background-color: #262626c9;
-  width: 55%;
-}
-.text{
-  background-color: #121212;
-  border: 0.2rem solid black;
-  color: #FFC529;
-  padding: 1rem 1rem;
-  margin-top: 5rem;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  align-content: center;
-  box-shadow: rgba(0, 0, 0, 0.25) 0 3.4rem 3.5rem, rgba(0, 0, 0, 0.12) 0 -0.75rem 1.8rem, rgba(0, 0, 0, 0.12) 0 1rem 1.4rem, rgba(0, 0, 0, 0.17) 0 2rem 1.8rem, rgba(0, 0, 0, 0.09) 0 -0.8rem 1rem;
-}
-.socials > a{
-  font-size: 1.8rem;
-  padding-left: 1rem;
-  color:#FFC529;
-}
-.socials >a:hover{
-  color:#af8206;
-  cursor: pointer;
-  transition: all 0.3s smooth;
+@import url("https://fonts.googleapis.com/css2?family=Inder&display=swap");
+
+:root {
+	--primaryYellow: #ffc529;
+	--secondayYellow: #feb74c;
+	--white: #ffffff;
+	--black: #000000;
+	--mainBackground: #232323;
+}
+
+body {
+	background-color: var(--mainBackground);
+	font-family: "Inder", "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+		"Lucida Sans", Arial, sans-serif;
+	padding: 1rem;
+	font-size: 16px;
+}
+
+.navbar {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	align-items: center;
+	gap: 10px;
+	color: var(--primaryYellow);
+}
+.navbar img {
+	width: 60px;
+	height: 33px;
+}
+.navbar h2 {
+	font-style: normal;
+	font-weight: 400;
+	font-size: 40px;
+	line-height: 50px;
+	display: flex;
+	align-items: center;
+	text-align: center;
+	margin: 0;
+}
+
+/**************
+**** MAIN ****
+**************/
+.main {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	align-items: center;
+}
+.main-text {
```

Comment:
what do you think about adding in something like  https://travishorn.com/responsive-grid-in-2-minutes-with-css-grid-layout-4842a41420fe this for the resposiveness? I think it would be best to have a responsive layout, rather than adding in a lot of different media queries. You can make the whole layout a grid, and then when viewport screen resizes, the grid moves around to fitthe viewport. 

OR you can use bootstrap!  https://getbootstrap.com/docs/5.0/getting-started/introduction/

Reply:
I will look into this, thanks for adding the resources!